From 236c77aa74cd7e1bf2c69e803a1395956aad98fd Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 17 Nov 2023 15:33:34 +0100
Subject: [PATCH] dt-bindings: nvmem: add new stm32mp25 compatible for
 stm32-romem

Add a new compatible for stm32mp25 support.

[Backport of commit d062d18d0e30 ("dt-bindings: nvmem: add new stm32mp25
 compatible for stm32-romem")]

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Acked-by: Conor Dooley <conor.dooley@microchip.com>
Signed-off-by: Srinivas Kandagatla <srinivas.kandagatla@linaro.org>
Change-Id: I89d2846974c23aae951332201ac0b9404f652637
---
 Documentation/devicetree/bindings/nvmem/st,stm32-romem.yaml | 1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/devicetree/bindings/nvmem/st,stm32-romem.yaml
+++ b/Documentation/devicetree/bindings/nvmem/st,stm32-romem.yaml
@@ -24,6 +24,7 @@ properties:
       - st,stm32f4-otp
       - st,stm32mp13-bsec
       - st,stm32mp15-bsec
+      - st,stm32mp25-bsec
 
   reg:
     maxItems: 1
